internal/subshell/bash/bash.go

```diff
@@ -132,6 +142,14 @@ func (v *SubShell) SetupShellRcFile(targetDir string, env map[string]string, nam
 
 // SetEnv - see subshell.SetEnv
 func (v *SubShell) SetEnv(env map[string]string) {
+	if path, pathExists := env["PATH"]; pathExists && runtime.GOOS == "windows" {
+		if bashified, err := osutils.BashifyPathEnv(path); err == nil {
+			env["PATH"] = bashified
+		} else {
+			multilog.Error("Unable to bashify PATH: %v", err)
```

Comment:
Wouldn't this almost certainly cause issues if we just ignore without warning the user? I think we'll need to return this error, which sadly means updating all use of `SetEnv`, but I don't see any other way around that at the moment.

Reply:
Okay, I've refactored SetEnv to allow for throwing an error. The only questionable result is that subshell.New() cannot throw this error. However, at that point `PATH` is unmodified and an error should never be thrown. Also, whenever a runtime is activated, a new environment is set and that is where we can notify the user of an error. I've documented this in a comment in subshell.New().